### PR TITLE
ansible: depends on krb5 for Linuxbrew

### PR DESCRIPTION
**Pull request by DoomHammer**
- [X] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [X] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

-----

Error log:

```
Last 15 lines from /home/doomhammer/.cache/Homebrew/Logs/ansible/26.python:
creating build/temp.linux-x86_64-2.7
creating build/temp.linux-x86_64-2.7/src
/home/doomhammer/.linuxbrew/bin/gcc-5 -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -Os -w -pipe -march=native -isystem/home/doomhammer/.linuxbrew/include -fPIC -I/home/doomhammer/.linuxbrew/include -I/home/doomhammer/.linuxbrew/opt/openssl/include -I/home/doomhammer/.linuxbrew/opt/sqlite/include -I/home/doomhammer/.linuxbrew/Cellar/python/2.7.11/include/python2.7 -c src/base64.c -o build/temp.linux-x86_64-2.7/src/base64.o
/home/doomhammer/.linuxbrew/bin/gcc-5 -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -Os -w -pipe -march=native -isystem/home/doomhammer/.linuxbrew/include -fPIC -I/home/doomhammer/.linuxbrew/include -I/home/doomhammer/.linuxbrew/opt/openssl/include -I/home/doomhammer/.linuxbrew/opt/sqlite/include -I/home/doomhammer/.linuxbrew/Cellar/python/2.7.11/include/python2.7 -c src/kerberos.c -o build/temp.linux-x86_64-2.7/src/kerberos.o
/home/doomhammer/.linuxbrew/bin/gcc-5 -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -Os -w -pipe -march=native -isystem/home/doomhammer/.linuxbrew/include -fPIC -I/home/doomhammer/.linuxbrew/include -I/home/doomhammer/.linuxbrew/opt/openssl/include -I/home/doomhammer/.linuxbrew/opt/sqlite/include -I/home/doomhammer/.linuxbrew/Cellar/python/2.7.11/include/python2.7 -c src/kerberosbasic.c -o build/temp.linux-x86_64-2.7/src/kerberosbasic.o
/home/doomhammer/.linuxbrew/bin/gcc-5 -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -Os -w -pipe -march=native -isystem/home/doomhammer/.linuxbrew/include -fPIC -I/home/doomhammer/.linuxbrew/include -I/home/doomhammer/.linuxbrew/opt/openssl/include -I/home/doomhammer/.linuxbrew/opt/sqlite/include -I/home/doomhammer/.linuxbrew/Cellar/python/2.7.11/include/python2.7 -c src/kerberosgss.c -o build/temp.linux-x86_64-2.7/src/kerberosgss.o
/home/doomhammer/.linuxbrew/bin/gcc-5 -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -Os -w -pipe -march=native -isystem/home/doomhammer/.linuxbrew/include -fPIC -I/home/doomhammer/.linuxbrew/include -I/home/doomhammer/.linuxbrew/opt/openssl/include -I/home/doomhammer/.linuxbrew/opt/sqlite/include -I/home/doomhammer/.linuxbrew/Cellar/python/2.7.11/include/python2.7 -c src/kerberospw.c -o build/temp.linux-x86_64-2.7/src/kerberospw.o
creating build/lib.linux-x86_64-2.7
/home/doomhammer/.linuxbrew/bin/gcc-5 -pthread -shared -L/home/doomhammer/.linuxbrew/lib -Wl,--dynamic-linker=/home/doomhammer/.linuxbrew/lib/ld.so -Wl,-rpath,/home/doomhammer/.linuxbrew/lib -L/home/doomhammer/.linuxbrew/lib -Wl,--dynamic-linker=/home/doomhammer/.linuxbrew/lib/ld.so -Wl,-rpath,/home/doomhammer/.linuxbrew/lib -Os -w -pipe -march=native -isystem/home/doomhammer/.linuxbrew/include build/temp.linux-x86_64-2.7/src/base64.o build/temp.linux-x86_64-2.7/src/kerberos.o build/temp.linux-x86_64-2.7/src/kerberosbasic.o build/temp.linux-x86_64-2.7/src/kerberosgss.o build/temp.linux-x86_64-2.7/src/kerberospw.o -L/home/doomhammer/.linuxbrew/lib -L/home/doomhammer/.linuxbrew/opt/openssl/lib -L/home/doomhammer/.linuxbrew/opt/sqlite/lib -L/home/doomhammer/.linuxbrew/Cellar/python/2.7.11/lib -lpython2.7 -o build/lib.linux-x86_64-2.7/kerberos.so -Wl,-Bsymbolic-functions -Wl,-z,relro -lgssapi_krb5 -lkrb5 -lk5crypto -lcom_err
/home/doomhammer/.linuxbrew/bin/ld: cannot find -lgssapi_krb5
/home/doomhammer/.linuxbrew/bin/ld: cannot find -lkrb5
/home/doomhammer/.linuxbrew/bin/ld: cannot find -lk5crypto
/home/doomhammer/.linuxbrew/bin/ld: cannot find -lcom_err
collect2: error: ld returned 1 exit status
error: command '/home/doomhammer/.linuxbrew/bin/gcc-5' failed with exit status 1
```

### CODE REVIEW

**Comment by sjackman** [comment]
`error: libxml/xmlversion.h: No such file or directory`

File: Formula/ansible.rb [hunk 1]
```diff
@@ -23,6 +23,9 @@ class Ansible < Formula
   depends_on "libyaml"
   depends_on "openssl"
   depends_on "libffi" unless OS.mac?
+  depends_on "homebrew/dupes/krb5" unless OS.mac?
+  depends_on "libxml2" unless OS.mac?
+  depends_on "libxslt" unless OS.mac?
```
Comment (rwhogg) [inline Formula/ansible.rb]: The dependency on `libxml2` is probably unnecessary, since `libxslt` depends on it too.

Comment (DoomHammer) [inline Formula/ansible.rb]: True, true :)

**Comment by DoomHammer** [comment]
Fixed

**Pull request closed without merging**